lib/components/atoms/Button/Button.style.js

```diff
@@ -25,6 +25,23 @@ const ButtonStyles = css`
     outline: none;
   }
 
+  &.icon {
```

Comment:
Create a variation, Icon-only or with-icon that styles a button with No text but just the Icon.